### ISSUE TITLE
FormulaGrader throws an error when passed an empty formula

### ISSUE THREAD

**Issue by jolyonb** [mitodl/mitx-grading-library#9]
This gives an error rather than just grading incorrect.

````
from mitxgraders import *
grader = FormulaGrader(
    answers="1"
)
print grader(None, "")
````

**Comment by ChristopherChudzicki** [comment]
Sorry I didn’t comment on this when you first posted, I just saw it referenced in #13.

I’mThinkingAboutThisNow, andIsAnErrorOnEmptyStreamsSubmission ofG sorry I didn’t comment on this when you first posted, I just saw it  referenced in #13.

 I’m thing about this now, and is this actually undesirable?  Might it not be better to universally raise an error on empty string submissions (For all graders, with useful error message and perhaps a configuration key override). Also isn’t that what edx does by default now?  I believe the rationale is to not use up student attempts if they accidentally submit an empty string.

**Comment by jolyonb** [comment]
When you have a large number of attempts and a two-part-in-one question, it's common to submit the first part of the question, leaving the second blank. This is the situation in which I noticed this error. I think the only way to deal with this is to mark an empty formula incorrect. Furthermore, edX won't let you submit an empty box if that's the only part to the question.

**Comment by ChristopherChudzicki** [comment]
Oh perfect— I had forgotten that the submit button was disabled rather than just raising an error. This sounds good then.  I can review this and other PRs tonight after dinner or tomorrow.

**Comment by ChristopherChudzicki** [comment]
Resolved by #13
